apps/api/src/billing/controllers/stripe/stripe.controller.ts

```diff
@@ -150,4 +150,17 @@ export class StripeController {
     const response = await this.stripe.getCustomerTransactions(currentUser.stripeCustomerId, options);
     return { data: response };
   }
+
+  @Protected([{ action: "read", subject: "StripePayment" }])
+  async exportTransactionsCsvStream(options: { startDate: string; endDate: string; timezone: string }): Promise<AsyncIterable<string>> {
+    const { currentUser } = this.authService;
+
+    if (!currentUser.stripeCustomerId) {
+      return (async function* () {
+        yield "Payments are not configured. Please start with a trial first";
```

Comment:
I'm not sure i follow. this should be an error. basically this call is forbidden 403 as user gotta do other stuff. we can also use 428 Precondition Required